src/main/java/org/example/Main.java

```diff
@@ -3,8 +3,10 @@
 public class Main {
     public static void main(String[] args) {
         System.out.println("Hello world!");
+        System.out.println("Hello world!");
```

Comment:
Hey, looks good but you might wanna use another greeting :)

Reply:
:-)